runtime/src/lib.rs

```diff
@@ -317,6 +321,21 @@ impl frame_system::Config for Runtime {
 	type PostTransactions = ();
 }
 
+pub struct TokenReleaseHandler;
+
+impl pallet_native_token_management::TokenReleaseHandler<u128> for TokenReleaseHandler {
```

Comment:
Just a mock handler that logs the data. It will be MN's responsibility to handle these events in their own ledger